cmd/libs/go2idl/client-gen/test_apis/testgroup.k8s.io/v1/types.go

```diff
@@ -25,8 +25,9 @@ import (
 
 type TestType struct {
 	unversioned.TypeMeta `json:",inline"`
-	api.ObjectMeta       `json:"metadata,omitempty"`
-	Status               TestTypeStatus `json:"status,omitempty"`
+	// +k8s:openapi-gen=false
+	api.ObjectMeta `json:"metadata,omitempty"`
```

Comment:
this is only a test type and adding unversioned objectMeta bring in a whole set of unversioned objects in the generate file that is not used anywhere (and honestly it was a pain to pinpoint). I decided to just exclude it, because this is a test type.